tests/playwright/shiny/inputs/test_input_selectize.py

```diff
@@ -13,27 +13,30 @@ def test_input_selectize_kitchen(page: Page, app: ShinyAppProc) -> None:
     expect(state.loc_label).to_have_text("Choose a state:")
     state.expect_label("Choose a state:")
 
-    state.expect_choices(["NY", "NJ", "CT", "WA", "OR", "CA", "MN", "WI", "IA"])
-    state.expect_choice_labels(
-        [
-            "New York",
-            "New Jersey",
-            "Connecticut",
-            "Washington",
-            "Oregon",
-            "California",
-            "Minnesota",
-            "Wisconsin",
-            "Iowa",
-        ]
-    )
-    state.expect_choice_groups(["East Coast", "West Coast", "Midwest"])
-
-    state.expect_selected("NY")
-    state.expect_multiple(False)
-
-    state.expect_width(None)
-
-    state.set("IA")
-
-    state.expect_selected("IA")
+    # TODO: This test was being run against input_select, not input_selectize
```

Comment:
This feels off to me. I'd like double check this in the morning before we merge the PR. 

Everything else is great!

Reply:
Yup. To my knowledge, selectize.js changed how it stores options / selected values. 

Either way, if we would have used the correct selectize testing class, the error would have been surfaced earlier. 

Yes, we should do this in a followup PR as the changes are not related to this Issue